scripts/clear-all-fault-leds.sh

```diff
@@ -52,19 +52,31 @@ done
 
 # Now, set the OperationalStatus Functional to what has been requested
 # We filter out the valid object path using a mapper call
+# If we find a unitX/coreX under then, we remove those because they are not mapped to LED's
+# Applicable for CPUs and DIMMs
 if [ ${#excluded_groups} -eq 0 ]
 then
     busctl call xyz.openbmc_project.ObjectMapper /xyz/openbmc_project/object_mapper xyz.openbmc_project.ObjectMapper \
     GetSubTreePaths sias "/xyz/openbmc_project/inventory" 0 1 "xyz.openbmc_project.State.Decorator.OperationalStatus" \
     | sed  's/ /\n/g' | tail -n+3 | awk -F "\"" '{print $2}' | while read -r line
     do
+        echo "$line" | grep "unit*\|core*" >/dev/null
```

Comment:
what is the '*' for?

Reply:
It is just a regular expression i used with grep

Reply:
ok, I think the point is the "*" should not be needed since it's always just "unit"? Putting a * there could cause some issue in the future where we introduce something new to the unit that we do not want to ignore.

Reply:
Done